app/helpers/sum.js

```diff
@@ -0,0 +1 @@
+export { default, sum } from 'ember-math-helpers/helpers/sum';
```

Comment:
@Mifrill I think we could just export from add here and be good to go. Something like this maybe?

```suggestion
export { default, add as sum } from 'ember-math-helpers/helpers/add';
```

Reply:
@rwwagner90 well, in this way needs to customize documentation somehow that provided by `ember-cli-addon-docs`, the original proposal is have documented `sum` helper:
![Selection_823](https://user-images.githubusercontent.com/23525618/134708961-ff4164c8-7e85-4b50-9b67-17aa744601ca.jpg)
maybe we can keep it consistent with the documentation addon?

Reply:
@rwwagner90 could you give an advice on how to customize documentation for this particular case?